### PR TITLE
Added Redis User

### LINKED ISSUE #165: Support Redis users
**Is this a request for help?**:

---

FEATURE REQUEST

We should be able to supply a `redis.user` value that sets the user for Redis. If it's not set you can default to the default user in the status quo.

### PR DESCRIPTION
**Pull request by EvanLiu0**
#### What this PR does / why we need it
Adds a Redis User to the connection string
- .

#### Which issue this PR fixes
Fixes #165 
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

#### Special notes for your reviewer

- .

#### Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Chart Version bumped
- [ ] Upgrading instructions are documented in the README.md

### CODE REVIEW

File: zammad/values.yaml [hunk 1]
```diff
@@ -3,7 +3,8 @@ image:
   # if not set appVersion field from Chart.yaml is used as default
   tag: ""
   pullPolicy: IfNotPresent
-  imagePullSecrets: []
+  imagePullSecrets:
+    []
```
Comment (monotek) [inline zammad/values.yaml:7]: please keep old format, also for all other lines.

**Review by monotek — CHANGES_REQUESTED** [review]
please also raise minor chart version

**Comment by AirP0WeR** [comment]
Hi @monotek kindly ask to add this feature to helm chart.

**Comment by MrGeneration** [comment]
This PR was closed due to inactivity.

**Pull request closed without merging**